aries-site/src/data/color.js

```diff
@@ -20,43 +20,43 @@ export const primaryColors = [
 export const supportingColors = [
   {
     name: 'teal',
-    value: 'teal',
-    hex: colors.teal,
+    value: 'teal!',
+    hex: colors['teal!'],
     rgb: 'rgb(0, 199, 129)',
     hsl: 'hsl(159, 100%, 39%)',
   },
   {
     name: 'blue',
-    value: 'blue',
-    hex: colors.blue,
+    value: 'blue!',
+    hex: colors['blue!'],
     rgb: 'rgb(0, 199, 129)',
     hsl: 'hsl(159, 100%, 39%)',
   },
   {
     name: 'purple',
-    value: 'purple',
-    hex: colors.purple,
+    value: 'purple!',
+    hex: colors['purple!'],
     rgb: 'rgb(0, 199, 129)',
     hsl: 'hsl(159, 100%, 39%)',
   },
   {
     name: 'red',
-    value: 'red',
-    hex: colors.red,
+    value: 'red!',
+    hex: colors['red!'],
```

Comment:
In the colors page, this renders as white text on red background and I get an accessibility error.

Reply:
But, in terms of updating the namespace, everything looks fine to me! My above comment might be a separate discussion about color values rather than namespace.